src/Chapter2.hs

```diff
@@ -501,7 +523,8 @@ True
 >>> isThird42 [42, 42, 0, 42]
 False
 -}
-isThird42 = error "isThird42: Not implemented!"
+isThird42 (_ : _ : 42 : _) = True
```

Comment:
It's always a good idea to write top-level function type signatures, don't forget about this :slightly_smiling_face: